public/css/index.css

```
@@ -19,6 +19,6 @@
 
 @layer components, legacy, paper;
 
-@import url('../styles.css?v=86')      layer(components);
-@import url('../antigravity.css?v=15') layer(legacy);
-@import url('paper.css?v=5')           layer(paper);
+@import url('../styles.css?v=87')      layer(components);
```

Stylelint (17.10.0)

[error] Expected "url('../styles.css?v=87')" to be "'../styles.css?v=87'" (import-notation)

(import-notation)